cmd/wio/commands/pac/user/logout.go

```diff
@@ -0,0 +1,27 @@
+package user
+
+import (
+    "errors"
+    "os"
+    "wio/cmd/wio/commands"
+    "wio/cmd/wio/log"
+    "wio/cmd/wio/utils/io"
+)
+
+func (cmd Logout) Execute() error {
+    dir, err := commands.GetDirectory(cmd)
+    if err != nil {
+        return nil
+    }
+    path := io.Path(dir, io.Folder, "token.json")
```

Comment:
this means you will have a different token based on your wio project. Can you log out if you delete that directory or change wio projects?

Reply:
Oops deleting the token means logging out

Reply:
isn't storing token this way a security risk? lol

Reply:
how do you think `npm`, `git`, `pip` and literally every other package manager does it?

the token has an expiry

Reply:
ik but the token is in `.wio` directory so it is only valid as long as I am in that specific wio project. Right?